goleveldb_test.go

```diff
@@ -1,43 +1,44 @@
+// Avoiding duplicate codes by lint, this implementation re-ordered functions
```

Comment:
Is this comment necessary information for future use? Or is it a description of this work?  If the latter, I suggest leaving it in this PR description or in a comment, not on the source.

Reply:
It's not necessary. Then, I'll remove the comments. thanks